components/PlantCard.js

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function PlantCard({ image, name, botanicalName }) {
+  return (
+    <PlantCardWrapper>
+        <PlantImageWrapper>
+      <StyledImage alt={name} src={image} fill="true"></StyledImage></PlantImageWrapper>
+      <h2> {name}</h2>
+      <p>{botanicalName}</p>
+     
+    </PlantCardWrapper>
```

Comment:
Just as an example.
The code is not well formatted here. Use Prettier (CMD + S) to format please. That makes it much easier to read 👍